test/lib/conn_parser_test.exs

```diff
@@ -1,5 +1,5 @@
 defmodule Xcribe.ConnParserTest do
-  use Xcribe.ConnCase, async: true
+  use Xcribe.ConnCase, async: false
```

Comment:
Why is this being changed? (Question really, I'm not saying it shouldn't, this trying to understand :sweat_smile:)

Reply:
Because in the `setup` block I'm changing some configurations, it has side effects in other tests, so it can run async...